libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m

```diff
@@ -440,7 +442,9 @@ - (SFRestRequest *)requestForUserInfo {
 
 - (SFRestRequest *)requestForVersions {
     NSString *path = @"/";
-    return [SFRestRequest requestWithMethod:SFRestMethodGET path:path queryParams:nil];
+    SFRestRequest* request = [SFRestRequest requestWithMethod:SFRestMethodGET path:path queryParams:nil];
```

Comment:
This API doesn't need an auth token. We use it in the new test below - but we needed SFRestAPI to know what authentication was not required.